source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -238,12 +209,13 @@ see the following API documentation:
 Summary
 -------
 
-In this guide, we explained how to convert data between BSON and POJO fields
-by performing the following:
+In this guide, we explain how to convert data between BSON and POJO fields
```

Comment:
S: since the style guide suggests limiting "we", you could reword to something like:
```suggestion
This guide describes how to convert data between BSON and POJO fields
```